package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fcp-sfd-frontend",
-  "version": "0.50.0",
+  "version": "0.51.2",
```

Comment:
Not specifically related to this PR but as a wider discussion point.  Unlike the FCP Platform where the version set here was extracted and used to version immutable artefacts for deployment, CDP does not follow this pattern.  Instead the version is determined post merge by CI.

That means you'll struggle to keep these versions in sync.  Maybe worth a discussion if you only want to capture major version changes here, not update it all (maybe reset to 0.0.0 if so) or something else.